docs/devcontainers.md

```diff
@@ -0,0 +1,164 @@
+# Devcontainer support
+
+A [development container](https://containers.dev) (or devcontainer for short) allows you to use a container as a development environment. It can be used to develop and run applications in the isolated environment that is persistent across different developers. This page describes the configuration for VS Code. You can refer to [this](https://containers.dev/supporting) page for other supported tools.
+
+## Requirements
+
+- Install [VS Code](https://code.visualstudio.com/docs/setup/setup-overview)
+- Install [Devcontainers Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+- Navigate to the root of Artery repository
+
+## Develop inside the container
+
+Working with devcontainers is very similar to [remote development](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh) using __ssh__.  After installing devcontainer extension, you should observe
+an icon in the status bar of your editor that looks like two arrowheads pointing at each other. Clicking on that
+icon reveals a menu for remote connections, which should include an option for opening current directory
+inside a container:
+
+`Reopen in Container`
+
+Selecting this entry brings you a list of options for containers. The ones Artery provides are called _arch-linux_ and _debian_.
+You can choose any of them, but keep in mind that graphic applications (including Artery) might fail due to version difference
```

Comment:
Different library versions should not be an issue, I would rather say something like "Sharing X11 displays between your host and the container environment may fail and require tweaking on your specific system."
Furthermore, I suggest replacing "graphic applications" (Artery is not about painting) with "graphical applications".